### PR TITLE
DAT-2540 edit mode styling for title and description

### PR DESCRIPTION
**Pull request by RafLeszczynski**

### CODE REVIEW

File: extensions/wikia/NjordPrototype/css/Njord.scss [hunk 1]
```diff
@@ -261,14 +261,21 @@
 	}
 
 	.title-wrap {
-		margin: 30px 330px 20px 30px;
+		margin: 0;
```
Comment (SebastianMarzjan) [inline extensions/wikia/NjordPrototype/css/Njord.scss]: We don't really need this line

File: extensions/wikia/NjordPrototype/css/Njord.scss [hunk 2]
```diff
@@ -261,14 +261,21 @@
...
+			margin: 0px 330px 0px 20px;
+			padding: 30px 0px 25px 10px;
+		}
+
 		.edit-box {
-			background-color: rgba(0, 0, 0, .03);
 			bottom: -64px;
 			display: none;
 			width: 100%;
+			text-transform: uppercase;
```
Comment (SebastianMarzjan) [inline extensions/wikia/NjordPrototype/css/Njord.scss]: Please move this before width (ordering)

**Comment by SebastianMarzjan** [comment]
Two small things and good to go

File: extensions/wikia/NjordPrototype/css/Njord.scss (hunk 1 above)
Comment (RafLeszczynski) [inline extensions/wikia/NjordPrototype/css/Njord.scss]: yes we do, unless we do refactoring of this stylesheet which I'd like to do in the next iteration of MoM project

Comment (SebastianMarzjan) [inline extensions/wikia/NjordPrototype/css/Njord.scss]: Can you elaborate? The following one seems to override all 4 values.

Comment (RafLeszczynski) [inline extensions/wikia/NjordPrototype/css/Njord.scss]: sorry this line shouldn't be there it was a leftover that I forgot to remove. My mistake, and good catch :+1:

**Pull request merged**